public/app/features/plugins/plugin_loader.ts

```diff
@@ -63,6 +63,9 @@ grafanaRuntime.SystemJS.config({
       authorization: true,
       loader: 'plugin-loader',
     },
+    '/public/*': {
+      authorization: false,
```

Comment:
This will break #9509 (access when grafana backend is behind a reverse proxy with auth)

Reply:
@xnyo if we consider this direction - can't we make this config only be set to false when the `plugins_cdn` is set to `redirect`? To not break #9509

Reply:
Yep. Hosted Grafana should be fine with `authorization: false`. I tried implementing that but I am not sure how to expose it to the frontend, so I may need some help from @jackw if we decide to use redirects.